skimage/draw/draw.py

```diff
@@ -712,8 +712,8 @@ def rectangle(origin, extent, shape=None):
 
     Returns
     -------
-    coords : (M, n) ndarray of int with the
-        Pixel coordinates of the nd rectangle.
+    coords : (M, N)
```

Comment:
Sounds good -- I sort of followed the polygon and ellipse examples. Would it make sense to change them so they read : 

Returns
    -------
    rr, cc : (N,) ndarray of int

ect..